### DISCUSSION TITLE
Budibase doesn't asked me for an API key

### DISCUSSION THREAD

**Discussion by george-bobby** [Budibase/budibase#1245] [Help] [answered]
Budibase doesn't ask me for an API key. I don't know whether it is working well or not. (This is my first time using budibase) And note that I downloaded budibase from official website

Here is a screenshot 

![Capture](https://user-images.githubusercontent.com/74462795/109909745-a21e4f00-7ccc-11eb-8d54-75f7ccf1acde.PNG)

**Comment by joebudi** [comment, marked as answer]
Hey! This is strange.
If you click into your app, then settings, then API key, could you check to see if there is one there, please. I have included a small video below to help:
https://www.loom.com/share/3eead0a20a71416898920cae6c134edd

**Reply by clcoffey** [reply]
This happened to me as well and when I followed the steps you outlined above there was no API key there. I hope this helps.

**Reply by EDMJohnny** [reply]
Same for me.   Just downloaded last night, running on Pop_Os! Linux (Ubuntu) - no key in program and no prompt to add one
